registry.json

```diff
@@ -3074,5 +3074,19 @@
       "telegram":"marin007",
       "operatorName":"Marian",
       "github":"aloopmarin007"
+  },
+  {
+      "ticker":"PEACH",
+      "poolId":"5df46b9796868647647f6d3a4ebd188920e70cd12720513337203b62",
+      "social":{
+            "instagram_handle": "peachpool.ada",
```

Comment:
Please can you put the correct key names here? E.g. remove `_handle` from them - please refer to other entries for examples.